Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.html

```diff
@@ -148,5 +148,22 @@ <h1>Edit Data Collector</h1>
             </div>
         </div>
     </div>
+    <div class="form-group">
+        <label for="status">Status</label>
+        <select class="form-control" id="status"
+                required
+                [(ngModel)]="changeBaseInformationCommand.status"
```

Comment:
There should be a seperate command for setting a data collector in / out of training. I can do that :)